debugging-with-ssh-access.md

```diff
@@ -0,0 +1,81 @@
+Often the best way to troubleshoot failures in your pipelines is to SSH into a
+job and inspect log files, running processes, and directory paths. Semaphore 2.0
+gives you the option to access all running jobs via SSH, to restart your jobs
+in debug mode, or to start on-demand virtual machines to explore the build
+environment.
+
+* [Explore the build environment](#explore-the-build-environment)
+* [Inspect the state of a running job](#inspect-the-state-of-a-running-job)
+* [Restart a job in debug mode](#restart-a-job-in-debug-mode)
+* [Port forward your web server and debug UI issues](#port-forward-your-web-server-and-debug-ui-issues)
+
```

Comment:
Maybe we can say *Exploring*, *Inspecting*, *Restarting* and *Port forwarding* here. Also, I think that port forwarding is not useful for web servers only but for all kinds of TCP applications.

Reply:
I agree that port-forwarding can used for much more then simple web server inspection. But, I am not sure if the high level overview is the best place to share this information. 

I want to give concrete examples to issues that arise during development and testing. The best example I have in mind is UI inspection. Everything else can be inspected in the machine with `tcpdump`, `netstat` and similar tools without port-forwarding.